src/CMakeLists.txt

```diff
@@ -26,8 +26,8 @@ target_link_libraries(h5z_zfp_static PUBLIC zfp::zfp ${HDF5_LIBRARIES})
 set_target_properties(h5z_zfp_static PROPERTIES POSITION_INDEPENDENT_CODE 1)
 set_target_properties(h5z_zfp_static PROPERTIES OUTPUT_NAME h5zzfp)
 install(TARGETS h5z_zfp_static
-  ARCHIVE DESTINATION lib
-  LIBRARY DESTINATION lib)
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
On Linux systems, the shared and static libs were put in lib64 and lib since it was hard coded to always install into lib for static.